src/almanack/reporting/cli.py

```diff
@@ -4,14 +4,14 @@
 
 import fire
 
-from almanack.reporting import report
+import almanack.processing.processing_repositories as whole_repo
```

Comment:
The naming here made me wonder if we might be planning to assess only parts of the repo (as opposed to all files as a "whole"). Double checking on the name selection: would it make sense to leave things as-is until we have something to differentiate how we process things?